// Saida esperada: Nenhum erro - exit code: 0

int func(){
  return 2.0;
}

int main(){
  func();
}